bin/README.md

```diff
@@ -13,14 +13,61 @@ Zone data can be managed in-memory, with flat files, or with an SQLite database.
 - ANAME resolution, for zone mapping aliases to A and AAAA records
 - Additionals section generation for aliasing record types
 
-## Optional protocol support
+## Cryptography provider
+
+The `hickory-dns` application supports choosing between two cryptography providers for
+protocols that require cryptography (DNSSEC, DNS-over-TLS, etc):
+
+1. [aws-lc-rs]
+2. [ring]
+
+Other feature flags are suffixed by the choice of provider. For example `tls-ring` or
+`tls-aws-lc-rs`.
+
+[aws-lc-rs]: https://crates.io/crates/aws-lc-rs
+[ring]: https://crates.io/crates/ring
+
+## DNSSEC
+
+In order to use DNSSEC you must enable a cargo feature corresponding to your 
+desired cryptography provider ([aws-lc-rs] or [ring]):
+
+* `dnssec-aws-lc-rs`
+* `dnssec-ring`
```

Comment:
Maybe protocols before DNSSEC? I have a feeling a lot more people are using the protocols.